### ISSUE TITLE
tableView didSelectRowAtIndexPath not being triggered

### ISSUE THREAD

**Issue by stephensilber** [StefanLage/SLParallaxController#2]
The tapGestureRecognizer added to the tableView causes the tableView didSelectRowAtIndexPath to not be called.

**Comment by bumaociyuan** [comment]
```
@interface SLParallaxController ()<UIGestureRecognizerDelegate>
```

```
self.tapTableViewGesture.delegate = self;
```

```
#pragma mark - UIGestureRecognizerDelegate
- (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldReceiveTouch:(UITouch *)touch {
    if (gestureRecognizer == self.tapTableViewGesture) {
        return _isShutterOpen;
    }
    return YES;
}
```

add these three parts in `SLParallaxController.m`

**Comment by StefanLage** [comment]
Thanks for the feedback ! Didn't notice that.
I merged bumaociyuan's solution bae692f987bd7d0d39f017c3054b2dd7f4b951a9
